src/prpy/planning/workspace.py

```diff
@@ -121,7 +124,7 @@ def PlanToEndEffectorOffset(self, robot, direction, distance,
         with robot:
             manip = robot.GetActiveManipulator()
             start_pose = manip.GetEndEffectorTransform()
-            traj = openravepy.RaveCreateTrajectory(self.env, '')
+            traj = openravepy.RaveCreateTrajectory(env, '')
```

Comment:
You should wrap `RetimeActiveTrajectory` in a `RobotStateSaver` that preserves the robot's `LinkTransformation`s, since the retimer may modify the robot's active DOF values.